PSICompile/ILCodeGen.cs

```diff
@@ -66,9 +107,36 @@ public class ILCodeGen : Visitor {
       }
       if (w.NewLine) Out ("    call void [System.Console]System.Console::WriteLine ()");
    }
-   
-   public override void Visit (NIfStmt f) => throw new NotImplementedException ();
-   public override void Visit (NForStmt f) => throw new NotImplementedException ();
+
+   public override void Visit (NIfStmt f) {
+      string labl1 = NextLabel (), labl2 = NextLabel ();
+      f.Condition.Accept (this);
+      Out ($"    brfalse {labl1}");
+      f.IfPart.Accept (this);
+      Out ($"    br {labl2}");
+      Out ($"   {labl1}:");
+      f.ElsePart?.Accept (this);
+      Out ($"   {labl2}:");

```

Comment:
Minor aesthetics: If there is no **else** part, you're generating 2 labels, when one would do.